core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.html

```diff
@@ -77,29 +77,46 @@
             </span>
           </a>
           <ul ngbDropdownMenu
-              class="dropdown-menu dropdown-menu oppia-navbar-dropdown"
+              class="dropdown-menu oppia-navbar-dropdown classroom-dropdown-menu"
               (mouseover)="openSubmenu($event, 'classroomMenu')"
               (mouseleave)="closeSubmenuIfNotMobile($event)"
               role="menu"
               autofocus>
             <li class="nav-item">
-              <a (keydown)="onMenuKeypress($event, 'classroomMenu', {shiftTab: ACTION_CLOSE})"
-                 (click)="navigateToClassroomPage('/learn/math')"
-                 class="dropdown-item nav-link oppia-navbar-tab-content protractor-test-classroom-link"
-                 href="#"
-                 [innerHTML]="'I18N_TOPNAV_CLASSROOM_BASIC_MATHS' | translate">
-              </a>
+              <div class="nav-item-main d-flex justify-content-between">
+                <div class="nav-item-left d-flex flex-column">
+                  <div class="nav-item-top d-flex justify-content-between">
+                    <div class="nav-content d-flex flex-column">
+                      <a (keydown)="onMenuKeypress($event, 'classroomMenu', {shiftTab: ACTION_CLOSE})" (click)="navigateToClassroomPage('/learn/math')" class="dropdown-item nav-link oppia-navbar-tab-content protractor-test-classroom-link nav-item-top-link" href="#" [innerHTML]="'I18N_TOPNAV_CLASSROOM_BASIC_MATHS' | translate"></a>
+                      <div class="des" [innerHTML]="'I18N_TOPNAV_CLASSROOM_DESCRIPTION_1' | translate"></div>
+                    </div>
+                    <picture>
+                      <source type="image/png"
+                              [srcset]="getStaticImageUrl('/topnav/learn.png')">
+                      <img [src]="getStaticImageUrl('/topnav/learn.png')"
+                           class="learn"
+                           alt="Learn" >
+                    </picture>
+                  </div>
+                  <div class="nav-item-bottom d-flex flex-column" *ngIf="!userIsLoggedIn">
+                    <div class="nav-item-links"  *ngFor="let topicSummary of classroomData.getTopicSummaries().slice(0, 3)">
```

Comment:
We will also have to check if any topic is present in the classroom, otherwise it gives console errors if classroom is enabled but there are no topics associated
![Screenshot from 2021-11-23 20-26-38](https://user-images.githubusercontent.com/68547101/143047833-309c772f-9602-483c-bffc-1a39a03da0d1.png)

Reply:
Done